exam/models/exam.py

```diff
@@ -271,6 +356,33 @@ class AdditionalExam(models.Model):
     weightage = fields.Char("WEIGHTAGE")
     create_date = fields.Date("Created Date", help="Exam Created Date")
     write_date = fields.Date("Updated date", help="Exam Updated Date")
+    color_name = fields.Integer("Color index of creator",
+                                compute='_compute_color_name', store=False)
+
+    @api.multi
+    def _compute_color_name(self):
+        for rec in self:
+            rec.color_name = rec.subject_id.id
+
+    @api.constrains('maximum_marks', 'minimum_marks')
+    def check_marks(self):
+        if self.minimum_marks > self.maximum_marks:
+            raise ValidationError(_('''Maximum marks should be greater
+                                    than minimum marks'''))
+
+    @api.model
+    def create(self, vals):
+        curr_dt = datetime.now()
+        new_dt = datetime.strftime(curr_dt, '%m/%d/%Y')
+        vals.update({'create_date': new_dt})
+        return super(AdditionalExam, self).create(vals)
```

Comment:
Cannot we set them in default? And why we set the field which is preset? the field is among 5 preset fields, isnt it?